robotsim.py

```diff
@@ -386,7 +297,14 @@ def __verifyPath(self,path): # path : ['L', 'R', 'U', 'D'...]
             return True
         return False
 
-    def debugTile(self):
+    def finishExploration(self):
+        self.broken = True
+        if self.col + self.row == 0:
+            print("El robot regresó a la base con éxito.")
+            self.points+=20
+        generate_map()
```

Comment:
If a robot reaches a fire or a collapsed zone, they might not be able to see the white screen with the failure message as this function might eventually be called. I think that it would help adding the:
`if not self.broken:`
to all functions that can generate the map. I only found this one and putOutFireFront to be relevant.

Reply:
This function will only be called to finish the program, so the robot will not move after this. The programmer will call this if he/she wants the +20 points al 0,0